CHANGELOG.md

```diff
@@ -61,6 +61,18 @@
 
 ### Dependencies
 
+### Dependencies
```

Comment:
Seems like we added another `### Dependencies` section

Reply:
oops! will add the fix in my PR for UPGRADE.md #6156